src/utils.js

```diff
@@ -217,3 +217,50 @@ function debug(level, varargs) {
     return console.log.apply(this, args);
   }
 }
+
+// inject a "follow this case on RECAP" button
+const recapButton = (court, pacerCaseId) => {
```

Comment:
Can we rename this as `createAlertButton` or something? recapButton vs. recapBanner has left me scratching my head a lot during this review.